src/systems/filecoin_vm/interpreter/vm_interpreter.go

```diff
@@ -226,31 +208,12 @@ func (vmi *VMInterpreter_I) ApplyMessage(
 	return
 }
 
-// Ensures a messages's receiving actor exists in the state tree.
-// If it doesn't, and the message is a simple value send to a pubkey-style address,
-// creates the receiver as an account actor in the returned state.
-func _ensureReceiver(tree st.StateTree, message msg.UnsignedMessage) (st.StateTree, bool) {
-	_, found := tree.GetActor(message.To())
-
-	if found {
-		return tree, true
-	}
-	if !message.To().IsKeyType() {
-		// Don't implicitly create an account actor for an address without an associated key.
-		return tree, false
-	}
-	if message.Method() != actor.MethodSend {
-		// Don't implicitly create account actor if message was expecting to invoke a method.
-		return tree, false
-	}
-	// TODO Create a new account actor via the InitActor, which will receive a new ID address
-	// and be placed in the state tree under that address.
-	// The init actor will maintain a map from pubkey address to ID address.
-	return tree, true
-}
-
 func _applyMessageBuiltinAssert(tree st.StateTree, message msg.UnsignedMessage, topLevelBlockWinner addr.Address) st.StateTree {
 	Assert(message.From().Equals(addr.SystemActorAddr))
+	// Note: this message CallSeqNum is never checked (b/c it's created in this file), but probably should be.
+	// Since it changes state, we should be sure about the state transition.
+	// Alternatively we could special-case the system actor and declare that its CallSeqNumber
+	// never changes (saving us the state-change overhead).
```

Comment:
We should probably do one or the other. I'd opt against special case but either seems good